db-scripts/src/main/resources/migration.sql

```diff
@@ -417,9 +418,11 @@ CREATE TABLE `reference_genome_gene` (
     FOREIGN KEY (`ENTREZ_GENE_ID`) REFERENCES `gene` (`ENTREZ_GENE_ID`) ON DELETE CASCADE
 );
 
-INSERT INTO reference_genome_gene (ENTREZ_GENE_ID, CYTOBAND, EXONIC_LENGTH, CHR, REFERENCE_GENOME_ID)
+INSERT INTO reference_genome_gene (ENTREZ_GENE_ID, GENETIC_ENTITY_ID,HUGO_GENE_SYMBOL, CYTOBAND, EXONIC_LENGTH, CHR, REFERENCE_GENOME_ID)
```

Comment:
can you add a space after the comma following `GENETIC_ENTITY_ID`?

Reply:
this remind me that GENETIC_ENTITY_ID is actually need to be removed.